carbonmark-api/test/routes/purchases/get.test.ts

```diff
@@ -1,37 +1,39 @@
 import { FastifyInstance } from "fastify";
-import { pick } from "lodash";
 import nock from "nock";
-import { aPurchase } from "../../../src/.generated/mocks/marketplace.mocks";
 import { GRAPH_URLS } from "../../../src/app.constants";
 import { Purchase } from "../../../src/models/Purchase.model";
+import carbonProjects from "../../fixtures/carbonProjects";
+import marketplace from "../../fixtures/marketplace";
 import { build } from "../../helper";
 import { DEV_URL, ERROR } from "../../test.constants";
 
-const mockPurchase = aPurchase();
-
-const responseFixture: Purchase = {
-  ...pick(mockPurchase, ["amount", "id", "price"]),
-  buyer: {
-    id: mockPurchase.user.id,
-  },
-  seller: {
-    id: mockPurchase.listing.seller.id,
-  },
+const purchaseModelFixture: Purchase = {
+  ...marketplace.purchase,
   listing: {
-    id: mockPurchase.listing.id,
+    id: marketplace.purchase.listing.id,
     project: {
-      ...pick(mockPurchase.listing.project, [
-        "key",
-        "methodology",
-        "name",
-        "projectID",
-        "vintage",
-      ]),
-      country: mockPurchase.listing.project?.country?.id || "",
+      key: marketplace.purchase.listing.project.key,
+      vintage: marketplace.purchase.listing.project.vintage,
+      country: carbonProjects.carbonProject.country!,
+      name: carbonProjects.carbonProject.name!,
+      methodology: carbonProjects.carbonProject.methodologies?.[0]?.id!,
+      projectID: carbonProjects.carbonProject.registryProjectId!,
```

Comment:
Should we handle the possibility of these being null or is it a non-issue?

Reply:
the generated mocks types are wrong for some reason 🤷 should not be null